src/guest/start.c

```diff
@@ -122,6 +122,27 @@ static int is_vfio_driver(const char *driver)
 	return -1;
 }
 
+static int remove_sof_tgl_snd_module()
+{
+	int pid;
+	int wst;
+	pid = fork();
+	if (pid == -1) {
+		fprintf(stderr, "%s: Failed to fork.", __func__);
+		return -1;
+	} else if (pid == 0) {
+		execlp("rmmod", "rmmod", "snd-sof-pci-intel-tgl", NULL);
```

Comment:
Is this solution working for all platforms: TGL, CML, ADL? do you need to distinguish different platforms?